app-pda/usbmuxd/usbmuxd-1.1.1_p20240915.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools systemd udev
+
+MY_COMMIT=0b1b233b57d581515978a09e5a4394bfa4ee4962
+
+DESCRIPTION="USB multiplex daemon for use with Apple iPhone/iPod Touch devices"
+HOMEPAGE="https://libimobiledevice.org/"
+SRC_URI="https://github.com/libimobiledevice/usbmuxd/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}"/${PN}-${MY_COMMIT}
+
+# src/utils.h is LGPL-2.1+, rest is found in COPYING*
+LICENSE="|| ( GPL-2 GPL-3 ) LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
+IUSE="selinux systemd"
+
+DEPEND="
+	acct-user/usbmux
```

Comment:
It is unrelated to the change you're making here, but I happened to notice this anyway -- it looks like this acct-user package could be in RDEPEND instead, as it isn't used during any `src_*` functions?